DEBUG 2022-12-11 12:20:06,850 wrapper.py Вызвана функция arg_parser с аргументами () {}
INFO 2022-12-11 12:20:13,557 client.py Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8888, имя пользователя: Sergey
INFO 2022-12-11 12:20:13,594 transport.py Попытка подключения № 1
INFO 2022-12-11 12:20:14,595 transport.py Попытка подключения № 2
INFO 2022-12-11 12:20:15,596 transport.py Попытка подключения № 3
INFO 2022-12-11 12:20:16,597 transport.py Попытка подключения № 4
INFO 2022-12-11 12:20:17,599 transport.py Попытка подключения № 5
CRITICAL 2022-12-11 12:20:18,600 transport.py Не удалось установить соединение с сервером
DEBUG 2022-12-11 12:24:48,401 wrapper.py Вызвана функция arg_parser с аргументами () {}
INFO 2022-12-11 12:24:53,203 client.py Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8888, имя пользователя: Sergey
INFO 2022-12-11 12:24:53,235 transport.py Попытка подключения № 1
INFO 2022-12-11 12:24:54,236 transport.py Попытка подключения № 2
INFO 2022-12-11 12:24:55,237 transport.py Попытка подключения № 3
INFO 2022-12-11 12:24:56,238 transport.py Попытка подключения № 4
INFO 2022-12-11 12:24:57,240 transport.py Попытка подключения № 5
CRITICAL 2022-12-11 12:24:58,241 transport.py Не удалось установить соединение с сервером
DEBUG 2022-12-11 12:34:26,399 wrapper.py Вызвана функция arg_parser с аргументами () {}
INFO 2022-12-11 12:34:31,235 client.py Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8888, имя пользователя: qwerty
INFO 2022-12-11 12:34:31,264 transport.py Попытка подключения № 1
INFO 2022-12-11 12:34:32,265 transport.py Попытка подключения № 2
INFO 2022-12-11 12:34:33,266 transport.py Попытка подключения № 3
INFO 2022-12-11 12:34:34,268 transport.py Попытка подключения № 4
INFO 2022-12-11 12:34:35,269 transport.py Попытка подключения № 5
CRITICAL 2022-12-11 12:34:36,270 transport.py Не удалось установить соединение с сервером
DEBUG 2022-12-11 12:41:07,494 wrapper.py Вызвана функция arg_parser с аргументами () {}
INFO 2022-12-11 12:41:12,124 client.py Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8888, имя пользователя: qwerty
INFO 2022-12-11 12:41:12,156 transport.py Попытка подключения № 1
DEBUG 2022-12-11 12:41:12,156 transport.py Установлено соединение с сервером
DEBUG 2022-12-11 12:41:12,156 transport.py Сформировано presence сообщение для пользователя qwerty
CRITICAL 2022-12-11 12:41:17,160 transport.py Потеряно соединение с сервером!
DEBUG 2022-12-11 12:42:04,793 wrapper.py Вызвана функция arg_parser с аргументами () {}
INFO 2022-12-11 12:42:11,454 client.py Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 7777, имя пользователя: qwerty
INFO 2022-12-11 12:42:11,480 transport.py Попытка подключения № 1
INFO 2022-12-11 12:42:12,481 transport.py Попытка подключения № 2
INFO 2022-12-11 12:42:13,482 transport.py Попытка подключения № 3
INFO 2022-12-11 12:42:14,484 transport.py Попытка подключения № 4
INFO 2022-12-11 12:42:15,485 transport.py Попытка подключения № 5
CRITICAL 2022-12-11 12:42:16,486 transport.py Не удалось установить соединение с сервером
DEBUG 2022-12-11 12:48:33,332 wrapper.py Вызвана функция arg_parser с аргументами () {}
INFO 2022-12-11 12:48:37,109 client.py Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 7777, имя пользователя: qwerty
INFO 2022-12-11 12:48:37,131 transport.py Попытка подключения № 1
DEBUG 2022-12-11 12:48:37,131 transport.py Установлено соединение с сервером
DEBUG 2022-12-11 12:48:37,131 transport.py Сформировано presence сообщение для пользователя qwerty
CRITICAL 2022-12-11 12:48:42,132 transport.py Потеряно соединение с сервером!
DEBUG 2022-12-11 12:52:08,292 wrapper.py Вызвана функция arg_parser с аргументами () {}
INFO 2022-12-11 12:52:12,894 client.py Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 7777, имя пользователя: qwery
INFO 2022-12-11 12:52:12,942 transport.py Попытка подключения № 1
DEBUG 2022-12-11 12:52:12,943 transport.py Установлено соединение с сервером
DEBUG 2022-12-11 12:52:12,943 transport.py Сформировано presence сообщение для пользователя qwery
CRITICAL 2022-12-11 12:52:17,943 transport.py Потеряно соединение с сервером!
